### ISSUE TITLE
Final Paper

### ISSUE THREAD

**Issue by mkduer** [mkduer/reinforcement-learning-snake-game#36]
- [ ] Editing:
   - [x] learning rate #64 - Chris -- editing
   - [x] discount factor #74 - Chris -- editing
   - [ ] Overall - Michelle
- [ ] Graphs: add last (this can be annoying to add early with shared Google dox)

---
**DONE**
- [x] Intro - Michelle #42 
- [x] Terminology - Chris
- [x] Implementation details - Michelle
- [x] Default hyperparameters for tests - Michelle
- [x] Training vs testing - Michelle
- [ ] Experimentation and Evaluation
   - [x] rewards #73 - Michelle
   - [x] epsilon #34 - Michelle
- [ ] Extra tests #76
  - [x] given extra time run 1,000,000+ episodes to see just how well the snake can learn (e.g. in terms of peak score, in terms of game converging) - Michelle
  - [x] test vs training: run test on constant hyperparameters multiple times to check for the consistency and overfitting (note append to file for this), possibly add with a changed board size - Michelle
  - [x] adjusting board size (larger, smaller, regular) - Michelle
- [x] Conclusion -- what more would we experiment #33, what would we change, la di da - Michelle
